pkg/urn/urn.go

```diff
@@ -0,0 +1,65 @@
+package urn
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+)
+
+var ErrInvalidURN = errors.New("invalid URN")
+
+const Namespace = "opni"
```

Comment:
Nit : consider renaming this to `Scope` or something along those lines

Reply:
Namespace is the idiomatic URN name for it

Reply:
ah right, its such an overloaded term